src/components/value-list/utils.ts

```diff
@@ -0,0 +1,56 @@
+import { getItemIndex } from "../pick-list/shared-list-logic";
+import { DragStatus } from "./interfaces";
+import { ValueList } from "./value-list";
+
+export function getScreenReaderText(
+  item: HTMLCalciteValueListItemElement,
+  status: DragStatus,
+  valueList: ValueList<HTMLCalciteValueListItemElement>
+): string {
+  const {
+    items,
+    intlDragHandleStart,
+    intlDragHandleActivated,
+    intlDragHandleNewPosition,
+    intlDragHandleCurrentPosition
+  } = valueList;
+  const total = items.length;
+  const position = getItemIndex(valueList, item) + 1;
+  switch (status) {
+    case "start":
+      return intlDragHandleStart
+        ? intlDragHandleStart
+        : `${item.label} ,press space and use arrow keys to re-order content. current position ${position} of ${total}`;
```

Comment:
Can you try switching this to an if/else statement? Switch statements can add a lot of unnecessary nesting.

Also, IMO, a newline between the statement and const assignment can help improve visually scanning the code.